java/client/src/main/java/glide/api/models/commands/RangeOptions.java

```diff
@@ -307,12 +310,29 @@ public static String[] createZRangeArgs(
         return arguments;
     }
 
+    public static GlideString[] createZRangeArgsBinary(
```

Comment:
Please make sure that `createZRangeArgs` calls `createZRangeArgsBinary` to avoid code duplication